tests/FINDOLOGIC/Export/Tests/XmlSerializationTest.php

```diff
@@ -469,6 +469,14 @@ public function testAddingInvalidUrlToImageElementCausesException(): void
         $image->getDomSubtree(new DOMDocument());
     }
 
+    public function testAddingImproperEncodedUrlToImageElementCausesException(): void
+    {
+        $this->expectException(InvalidUrlException::class);
```

Comment:
Also make sure that the exception message is right, because this test will probably pass with and without the non-test change.